templates/geosafe/analysis/dynamic_scripts/create/side_panel.html

```diff
@@ -218,7 +219,7 @@
                 hazard_exposure_rect_groups.clearLayers();
             }
 
-            show_info_text('InaSAFE Exposure', $a.text() + ' selected.');
+            show_info_text(info_text_title, $a.text() + ' {% trans 'selected.' %}');
```

Comment:
Use javascript string template literal interpolation if possible.

```suggestion
            show_info_text(info_text_title, ' {% trans '${$a.text()} selected.' %}');
```